### PR TITLE
Feature/bocm 486 create file storage service

### PR DESCRIPTION
**Pull request by mayya-bond-kc**
## Describe your changes

- creates new service for azure blob storage integration

PS I know this is missing some documentation and some endpoints (Dockerisation etc), but thought it would be better to start moving this into the develop branch sooner rather than later

## Type of change 🧩

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] Documentation update
- [ ] Other (please explain in the description section above)

## Checklist before requesting a review

- [x] I have performed a self-review of my own code
- [x] I have double checked this work does not include any hardcoded secrets or passwords
- [ ] I have made corresponding changes to the documentation
- [x] I have provided details on how I have tested my code
- [x] I have referenced the ticket number above
- [x] I have provided a description of the ticket
- [x] I have included unit tests to cover any testable code changes

### CODE REVIEW

File: apps/document-storage/.env.development [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+# Default : 3443
+HTTPS_PORT=3443
+
+# - - - - - - - - - Swagger - - - - - - - 
+
+SWAGGER_JSON_DIR=./src/server/swagger-output.json
+
+# - - - - - - - Blob Storage - - - - - -
+
+AZURE_BLOB_STORE_CONNECTION_STRING=DefaultEndpointsProtocol=http;AccountName=devstoreaccount1;AccountKey=Eby8vdM02xNOcqFlqUwJPLlmEtlCDXJ1OUzFT50uSRZ6IFsuFq2UVErCz4I6tq/K1SZFPTOtr/KBHBeksoGMGw==;BlobEndpoint=http://localhost:10000/devstoreaccount1;QueueEndpoint=http://localhost:10001/devstoreaccount1;
```
Comment (radum) [inline apps/document-storage/.env.development:22]: This key should not be here.

Comment (radum) [reply apps/document-storage/.env.development:22]: Any secret should be set in the gitignored file. Here is just for reference.

Comment (mayya-bond-kc) [reply apps/document-storage/.env.development:22]: This secret key actually not secret at all. This is the same secret key that everyone gets when using the local version of the azure blob storage. Here is a public page with all this info: https://docs.microsoft.com/en-us/azure/storage/common/storage-configure-connection-string

In the dev/test envs, we will just need to override this connection string.

**Review by francescobrenci-AR — APPROVED** [review]

**Pull request merged**